commons/src/main/res/values-da/strings.xml

```diff
@@ -265,11 +265,11 @@
     <string name="password_protect_whole_app">Beskyt hele appen med en adgangskode</string>
     <string name="keep_last_modified">Kopi/flyt/omdøb skal ikke påvirke ændringstidspunktet</string>
     <string name="show_info_bubble">Vis en info-boble når jeg scroller med rullepanelet</string>
-    <string name="prevent_phone_from_sleeping">Forhindr telefonen i at falde i søvn når appen er i forgrunden</string>
+    <string name="prevent_phone_from_sleeping">Prevent phone from sleeping while the app is in foreground</string>
```

Comment:
guess you didn want to override this one?

Reply:
I do not understand, why not?

Reply:
Ha, sorry, no.